src/test/java/org/sonarsource/sonarlint/ls/mediumtests/LanguageServerMediumTests.java

```diff
@@ -121,6 +122,7 @@ void prepare() {
   public void mockSonarQube() {
     mockWebServerExtension.addStringResponse("/api/system/status", "{\"status\": \"UP\", \"version\": \"9.3\", \"id\": \"xzy\"}");
     mockWebServerExtension.addStringResponse("/api/authentication/validate?format=json", "{\"valid\": true}");
+    mockWebServerExtension.addProtobufResponse("/api/components/search.protobuf?qualifiers=TRK&ps=500&p=1", Components.SearchWsResponse.newBuilder().build());
```

Comment:
What does this change do? 🤔

Reply:
It prepares an empty response to the "new code period" request, which silences a noisy exception log during tests. It's not related to DBD, more like housekeeping 🧹